lib/rubocop/cop/lint/literal_in_interpolation.rb

```diff
@@ -21,7 +21,7 @@ class LiteralInInterpolation < Base
         MSG = 'Literal interpolation detected.'
         COMPOSITE = %i[array hash pair irange erange].freeze
 
-        def on_interpolation(begin_node)
+        def on_interpolation(begin_node) # rubocop:disable Metrics/AbcSize
```

Comment:
`rubocop:disable Metrics/AbcSize` may be resolved at some point in the future. To avoid polluting `git blame` when that happens, it would be better to place new annotations before and after the method, rather than inline:

```ruby
# rubocop:disable Metrics/AbcSize
def on_interpolation(begin_node)
  ...
end
# rubocop:enable Metrics/AbcSize
```